Pearson-Advance/tutor-build-image-action
# PADV-328 - Add the Tutor Pearson plugin to the Action.

## Ticket

https://agile-jira.pearson.com/browse/PADV-328

## Description

We created a brand new Tutor plugin containing custom definitions for our Pearson VUE installation. We need to include this Tutor plugin in the image-building process so that the Tutor images have our custom changes and definitions.

## Type of Change

- [x] Add `install-tutor-pearson-plugin` to determine if the plugin will be installed with the action.
- [x] add installation plugin using the steps from tutor to install private repositories using pip.
- [x] Run `tutor plugins enable pearson-plugin` and `tutor config save`.

## How to test

- Create a new workflow action in another repository
- In that action use this action with `uses: Pearson-Advance/tutor-build-image-action@vue/PADV-328 `
- Pass the necessary inputs to build a plugin image
- In the following job you can see that the steps run correctly: https://github.com/Pearson-Advance/edx-platform/actions/runs/4430083924/jobs/7771363566

![image](https://user-images.githubusercontent.com/30726391/225106777-641db392-2224-49de-baae-589062adb861.png)

## Reviewers
